### ISSUE TITLE
osm11 injoignable

### ISSUE THREAD

**Issue by cquest** [osm-fr/infrastructure#333]
Depuis minuit environ osm11 n'est pas joignable.

Principaux services impactés (coché si à nouveau disponible):
- [x] tuiles cyclosm
- [x] bano.openstreetmap.fr
- [x] demo.openstreetmap.fr

Message envoyé à notre contact fondation free pour avoir un retour, rebooter le serveur.

**Comment by cquest** [comment]
Tuiles cyclosm redirigées temporairement vers celles d'osm.org

**Comment by TANK2003** [comment]
demo.openstreetmap.fr passe pas également, lié à OSM 11 également ?

**Comment by TANK2003** [comment]
c'est revenu apparemment, merci !

**Comment by alkino** [comment]
c'est reparti ?

**Comment by XandrexOSM** [comment]
> 
> 
> c'est revenu apparemment, merci !

c'est actuellement KO. 
![image](https://user-images.githubusercontent.com/92858966/142504543-f7da2c35-3c03-4753-8808-fc51c2d70630.png)

**Comment by cquest** [comment]
Ce n'était que temporaire malheureusement, le serveur a un problème hardware au niveau CPU ou carte mère et a replanté peu de temps après.
Je l'ai donc rapatrié pour voir si c'est réparable, c'est le deuxième serveur qui a ces symptômes.
En attendant, RAM/SSD/HDD ont été transplantés dans une autre machine pour que le service soit remis en route rapidement.

**Comment by cquest** [comment]
Tuiles CyclOSM de retour...

**Comment by cquest** [comment]
bano.openstreetmap.fr de retour partiel (problème de certificat)

**Comment by cquest** [comment]
demo.openstreetmap.fr lui aussi de retour partiel (même problème de certificat)

**Comment by cquest** [comment]
Certificats remis en place par @jocelynj

**Comment by vdct** [comment]
Je n'ai pas accès à osm162 pour l'instant en ssh / ipv6. Je ne sais pas si c'est un autre souci ou si ça fait parti du tunnel de réparations en cours...

**Comment by cquest** [comment]
J'ai rien touché sur la config IPv6... je laisse @jocelynj faire

**Comment by jocelynj** [comment]
J'ai normalement reconfiguré l'ipv6 comme il fallait, et bano.vm.openstreetmap.fr est accessible. @vdct, tu peux confirmer ?

**Comment by michaelblyons** [comment]
Working for me. 🆗

**Comment by vdct** [comment]
> J'ai normalement reconfiguré l'ipv6 comme il fallait, et bano.vm.openstreetmap.fr est accessible. @vdct, tu peux confirmer ?

Merci Jocelyn, ça fonctionne chez moi. L'IP v6 a changé par rapport à il y a quelques jours, je devrais peut-être me borner à utiliser bano.vm.openstreetmap.fr plutôt que l'ip directement, non ?

**Comment by cquest** [comment]
Oui, il est vraiment préférable d'utiliser une entrée DNS plutôt que l'IP qu'on peut toujours modifier (surtout une IPv6).

**Comment by vinber** [comment]
est-ce à fermer ici ?

**Comment by Marc-marc-marc** [comment]
pour mémoire certificat ssl date de septembre 2021 sachant qu'on le regen au moins tous les 2 mois... c'était ko à cause des sites migrés hors d'osm11 mais tjs présent dans hosts-list
est.ce que cela a été migré à la main et oublié ou migrer via ansible et faut chercher le bug ?
